beacon-chain/monitor/process_block.go

```diff
@@ -55,10 +56,32 @@ func (s *Service) processBlock(ctx context.Context, b block.SignedBeaconBlock) {
 	s.processAttestations(ctx, state, blk)
 }
 
+// updateSyncCommitteeTrackedVals updates the sync committee assignments of our
+// tracked validators. It gets called when we sync a block after the Sync Period changes.
+func (s *Service) updateSyncCommitteeTrackedVals(state state.BeaconState) {
```

Comment:
moved `updateSyncCommitteeTrackedVals` next to its only usage instead of `service.go`, felt cleaner and to seeing the locks are in place next to it

Reply:
This needs to be in service in the service launch, that's why I had it there. The next PR may change your mind about this one

Reply:
got it, we can move it back